packages/api/src/controllers/sensorController.js

```diff
@@ -174,7 +176,12 @@ const sensorController = {
         // Save sensors in database
         const sensorLocations = await Promise.all(
           registeredSensors.map(async (sensor) => {
-            return await SensorModel.createSensor(sensor, farm_id, user_id);
+            return await SensorModel.createSensor(
```

Comment:
If this fails, but all sensors are registered with ensemble, we are still sending a 200 response